client/dashboard/app/i18n.tsx

```diff
@@ -27,6 +30,26 @@ async function fetchLocaleData( language: string, signal: AbortSignal ) {
 	}
 }
 
+function getHtmlLangAttribute( i18n: I18n, fallback: string ) {
+	// translation of this string contains the desired HTML attribute value
+	const slug = i18n.__( 'html_lang_attribute' );
```

Comment:
This is necessary because the lang slug need by `<html>` is a different ISO standard from the one we use.

For an example locale where `'html_lang_attribute'` isn't translated try `en`.